ChangeLog.md

```diff
@@ -30,13 +30,13 @@
 - Renamed `runTraceAsOutput` to `traceToOutput`
 - Renamed `runTraceIO` to `traceToIO`
 - Renamed `sendM` to `embed` (thanks to @googleson78)
-- The `NonDet` effect will no longer perform effects in untaken branches (thanks to @KingoftheHomeless)
```

Comment:
To explain, that behaviour is specific to `runNonDetMaybe` and `nonDetToError`. `runNonDet` still function as it did before.

Reply:
makes sense